source/_components/notify.hipchat.markdown

```diff
@@ -0,0 +1,53 @@
+---
+layout: page
+title: "HipChat"
+description: "Instructions how to add HipChat notifications to Home Assistant."
+date: 2017-08-10 15:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: hipchat.png
+ha_category: Notifications
+ha_release: "0.51"
+---
+
+
+The `hipchat` platform allows you to deliver notifications from Home Assistant to [HipChat](https://hipchat.com/).
+
+You need to obtain a [HipChat API token](https://developer.atlassian.com/hipchat/guide/hipchat-rest-api/api-access-tokens#APIaccesstokens-Usergeneratedtokens) to be able to send notifications.
+
+To enable the HipChat notification in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+notify:
+  - name: NOTIFIER_NAME
+    platform: hipchat
+    token: ABCDEFGHJKLMNOPQRSTUVXYZ
+    room: '1234567'
```

Comment:
The API treats it as a string rather than a number, and I suppose there's the possibility that a room ID might be non-numeric (though I haven't seen any).